doc/API.md

```diff
@@ -84,3 +84,49 @@ This command does not take any parameter for now.
 | `amount`       | int           | Value of the UTxO in satoshis                                    |
 | `outpoint`     | string        | Transaction id and output index of this coin                     |
 | `block_height` | int or null   | Blockheight the transaction was confirmed at, or `null`          |
+
+
+### `createspend`
+
+Create a transaction spending one or more of our coins. All coins must exist and not be spent.
+
+Will error if the given coins are not sufficient to cover the transaction cost at 90% (or more) of
+the given feerate. If on the contrary the transaction is more than sufficiently funded, it will
+create a change output when economically rationale to do so.
+
+This command will refuse to create any output worth less than 5k sats.
+
+#### Request
+
+| Field          | Type              | Description                                                       |
+| -------------- | ----------------- | ----------------------------------------------------------------- |
+| `outpoints`    | list of string    | List of the coins to be spent, as `txid:vout`.                    |
+| `destinations` | object            | Map from Bitcoin address to value                                 |
```

Comment:
Should probably swap those so eventually we can make `outpoints` optional to use an automated coin selection.

Reply:
Created https://github.com/revault/minisafe/issues/33 for this.